bin/run.sh

```diff
@@ -44,5 +44,10 @@ echo "${DEPLOYMENT_DESCRIPTION}"
 # Record deployment using the New Relic Python Admin CLI
 newrelic-admin record-deploy "${NEW_RELIC_CONFIG_FILE}" "${DEPLOYMENT_DESCRIPTION}"
 
+# make sure python accepts the cloud.gov certificate authority file for proxy
```

Comment:
alternatively we could implement this as a cloud.gov [app task](https://docs.cloudfoundry.org/devguide/using-tasks.html) but I think it's a bit cleaner to embed in `run.sh` -- but open to discussion on this.

Reply:
I agree with this choice. I suspect it also makes app backup+recovery clearer if there's an outage and we need to reconstruct Tock.